src/app/ui/components/button/button.stories.tsx

```diff
@@ -1,9 +1,9 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { HStack, styled } from 'leather-styles/jsx';
 
-import { BtcIcon } from '../icons/btc-icon';
-import { ChevronDownIcon } from '../icons/chevron-down-icon';
-import { PlaceholderIcon } from '../icons/placeholder-icon';
+import { BtcIcon } from '../../icons/btc-icon';
```

Comment:
What do you think of using `@app/ui/icons....` instead of `../`

Reply:
Yep, for sure, I try to catch these when I do a find/replace. Thx, I'll fix them.